src/app/component/cbp-plan-stats/cbp-plan-stats.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="flex flex-col gap-4 mt-8 mb-8 my-cbp">
     <div class="flex flex-middle flex-between position-relative ">
         <p class="title">My iGOT</p>
-        <mat-icon mat-icon-button [matMenuTriggerFor]="menu"   class="ws-mat-default-text mat-icon notranslate material-icons mat-icon-no-color cursor-pointer">filter_list</mat-icon>
+        <!-- <mat-icon mat-icon-button [matMenuTriggerFor]="menu"   class="ws-mat-default-text mat-icon notranslate material-icons mat-icon-no-color cursor-pointer">filter_list</mat-icon>
```

Comment:
same will work for mobile screen as well?

Reply:
yes it is responsive
same thing will come